files/default/slurmctld.service

```diff
@@ -1,12 +1,13 @@
 [Unit]
 Description=Slurm controller daemon
-After=network.target munge.service
+After=network-online.target munge.service
+Wants=network-online.target
 ConditionPathExists=/opt/slurm/etc/slurm.conf
 
 [Service]
 Type=simple
 EnvironmentFile=-/etc/sysconfig/slurmctld
-ExecStart=/opt/slurm/sbin/slurmctld -D $SLURMCTLD_OPTIONS
+ExecStart=<%= node['cluster']['slurm']['install_dir'] %>/sbin/slurmctld -D -s $SLURMCTLD_OPTIONS
```

Comment:
In 3.x, we use `default['cluster']['slurm']['install_dir'] = "/opt/slurm"`, so it is `node['cluster']['slurm']['install_dir']` here, but in 2.11, the node attribute is `default['cfncluster']['slurm']['install_dir'] = "/opt/slurm"`, we need to use `node['cfncluster']['slurm']['install_dir']` instead.

Reply:
This file need to be renamed with end with.erb and then put in `/erbhttps://github.com/aws/aws-parallelcluster-cookbook/tree/release-2.11-slurm2108/templates/default/slurm`

Reply:
ok, I'll change that now